### ISSUE TITLE
California uprating doesn't work

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine-us#4151]
For example, the dependent exemption is uprated:
https://github.com/PolicyEngine/policyengine-us/blob/ca967aba24271ed471c722ef3684b05e5502c034/policyengine_us/parameters/gov/states/ca/tax/income/exemptions/dependent_amount.yaml#L2-L12

But it doesn't work [in the app](https://policyengine.org/us/policy?focus=gov.states.ca.tax.income.exemptions.dependent_amount):
<img width="896" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/6076111/dc04d912-9dc8-4cda-a8d5-9afa1687afdb">

California CPI itself is uprated by the CBO CPI forecast:

https://github.com/PolicyEngine/policyengine-us/blob/ca967aba24271ed471c722ef3684b05e5502c034/policyengine_us/parameters/gov/states/ca/cpi.yaml#L2-L10

That works [in the app](https://policyengine.org/us/policy?focus=gov.states.ca.cpi), at least through 2029:

<img width="838" alt="image" src="https://github.com/PolicyEngine/policyengine-us/assets/6076111/e123f4bc-651d-43a4-b74f-1f081187a37b">

This seems like a uprating-chaining issue for core but logging here for now.

**Comment by nikhilwoodruff** [comment]
Right, my guess is we need to add some logic specifying ordering of uprating parameters; right now we just traverse the tree and apply in that order, irrespective of dependency.

**Comment by MaxGhenis** [comment]
Filed https://github.com/PolicyEngine/policyengine-core/issues/292 for general uprating chaining
